docs/get-started-react.md

```diff
@@ -0,0 +1,262 @@
+import { SurveyCreatorComponent, SurveyCreator } from "survey-creator-react";
+import "survey-core/defaultV2.min.css";
+import "survey-creator-core/survey-creator-core.min.css";
+
+const creatorOptions = {
+  showLogicTab: true,
+  isAutoSave: true
+};
+
+export function SurveyCreatorWidget() {
+  const creator = new SurveyCreator(creatorOptions);
+  return (
+    <SurveyCreatorComponent creator={creator} />
+  )
+}
+```
+</details>
+
+## Save and Load Survey Model Definitions
+
+Survey Creator produces survey model definitions as JSON objects. You can persist these objects on your server: save updates and restore previously saved definitions. To save a JSON object, implement the `saveSurveyFunc` function. It accepts two arguments:
+
+- `saveNo`      
+An incremental number of the current change. Since web services are asynchronous, you cannot guarantee that the service receives the changes in the same order as the client sends them. For example, change #11 may arrive to the server faster than change #10. In your web service code, update the storage only if you receive changes with a higher `saveNo`.
+
+- `callback`        
+A callback function. Call it and pass `saveNo` as the first argument. Set the second argument to `true` or `false` based on whether the server applied or rejected the change.
+
+The following code shows how to use the `saveSurveyFunc` function to save a survey model definition in a <a href="https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage" target="_blank">`localStorage`</a> or in your web service:
+
+
+```js
+export function SurveyCreatorWidget() {
+  // ...
+  creator.saveSurveyFunc = (saveNo, callback) => { 
+    // If you use localStorage:
+    window.localStorage.setItem("survey-json", creator.text);
+    callback(saveNo, true);
+
+    // If you use a web service:
+    saveSurveyJson(
+        "https://your-web-service.com/",
+        creator.JSON,
+        saveNo,
+        callback
+    );
+  };
+  // ...
+}
+
+// If you use a web service:
+function saveSurveyJson(url, json, saveNo, callback) {
+  const request = new XMLHttpRequest();
+  request.open('POST', url);
+  request.setRequestHeader('Content-Type', 'application/json;charset=UTF-8');
+  request.addEventListener('load', () => {
+      callback(saveNo, true);
+  });
+  request.addEventListener('error', () => {
+      callback(saveNo, false);
+  });
+  request.send(JSON.stringify(json));
+}
+```
+
+To load a survey model definition JSON into Survey Creator, assign the definition to Survey Creator's [`JSON`](https://surveyjs.io/Documentation/Survey-Creator?id=surveycreator#JSON) or [`text`](https://surveyjs.io/Documentation/Survey-Creator?id=surveycreator#text) property. Use `text` if the JSON object is converted to string; otherwise, use `JSON`. The following code takes a survey model definition from the `localStorage`. If the definition is not found (for example, when Survey Creator is launched for the first time), a default JSON is used:
```

Comment:
```suggestion
To load a survey model definition JSON into Survey Creator, assign the definition to Survey Creator's [`JSON`](https://surveyjs.io/Documentation/Survey-Creator?id=surveycreator#JSON) or [`text`](https://surveyjs.io/Documentation/Survey-Creator?id=surveycreator#text) property. Use `text` if the JSON object is converted to a string; otherwise, use `JSON`. The following code takes a survey model definition from the `localStorage`. If the definition is not found (for example, when Survey Creator is launched for the first time), a default JSON is used:
```